ocollection.h

```diff
@@ -0,0 +1,90 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+struct OVector{
+    int size;
+    int capacity;
+    int* array;
+};
+
+void initializeVector(struct OVector* vector) {
+    vector->size = 0;
+    vector->capacity = 5;
+    vector->array = malloc(vector->capacity * sizeof(int));
+}
+
+int* createArray( int capacity) {
+    int* array = malloc(capacity+sizeof(int));
+    return array;
+}
+
+void deleteArray(struct OVector* vector) {
+    free(vector->array);
+}
+
+void resizeArray(struct OVector* vector, int newCapacity) {
+    int* newArray = createArray(newCapacity);
```

Comment:
`realloc` function can simplify this section of code (https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/realloc?view=msvc-170)